wcc-contentful/app/controllers/wcc/contentful/webhook_controller.rb

```diff
@@ -19,9 +18,10 @@ class WebhookController < ApplicationController
     end
 
     def receive
-      event = params.require('webhook').permit!
-      event.require('sys').require(%w[id type])
-      event = event.to_h
+      params.require('sys').require(%w[id type])
+      params.permit('sys', 'fields').permit!
```

Comment:
can you not just `to_h` the result of this?

Reply:
I apparently still need the `slice` line.